build/org.eclipse.birt.target/org.eclipse.birt.target.target

```diff
@@ -0,0 +1,112 @@
+			<unit id="org.eclipse.datatools.common.doc.user.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.datatools.connectivity.doc.user.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.datatools.intro.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.datatools.sqltools.doc.user.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.datatools.sqldevtools.data.feature.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.datatools.sqldevtools.ddl.feature.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.datatools.sqldevtools.schemaobjecteditor.feature.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.datatools.sqldevtools.sqlbuilder.feature.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.datatools.enablement.hsqldb.feature.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.datatools.enablement.oda.designer.feature.feature.group" version="0.0.0"/>
+		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<repository location="https://download.eclipse.org/tools/gef/updates/legacy/releases/4.0.0_gef-master_1952"/>
+			<unit id="org.eclipse.draw2d.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.gef.feature.group" version="0.0.0"/>
+		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<repository location="https://download.eclipse.org/webtools/downloads/drops/R3.13.0/R-3.13.0-20190308131645/repository/"/>
+			<unit id="org.eclipse.jst.enterprise_ui.feature.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.wst.jsdt.feature.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.jst.web_ui.feature.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.wst.web_ui.feature.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.jst.common.fproj.enablement.jdt.feature.group" version="0.0.0"/>
+		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<repository location="http://download.eclipse.org/jetty/updates/jetty-bundles-9.x/9.4.15.v20190215"/>
+			<unit id="org.eclipse.jetty.osgi.boot" version="0.0.0"/>
+			<unit id="org.eclipse.jetty.server" version="0.0.0"/>
+			<unit id="org.eclipse.jetty.util" version="0.0.0"/>
+			<unit id="org.eclipse.jetty.webapp" version="0.0.0"/>
+			<unit id="org.eclipse.jetty.plus" version="0.0.0"/>
+			<unit id="org.eclipse.jetty.servlet" version="0.0.0"/>
+			<unit id="org.eclipse.jetty.http" version="0.0.0"/>
+		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<unit id="org.apache.jasper" version="0.0.0"/>
+			<repository location="http://download.eclipse.org/gemini/updates/web/3.0.4"/>
```

Comment:
Let's always declare `repository` before any `unit`, like it is done above and below